<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>colorSettings</class>
 <widget class="QWidget" name="colorSettings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>736</width>
    <height>422</height>
   </rect>
  </property>
  <layout class="QVBoxLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="5" column="0">
      <widget class="QPushButton" name="schemeEditButton">
       <property name="toolTip">
        <string>Configure the current color scheme</string>
       </property>
       <property name="text">
        <string>Edit...</string>
       </property>
       <property name="flat">
        <bool>false</bool>
       </property>
       <property name="icon">
        <iconset theme="edit-entry">
         <normaloff/>
        </iconset>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QPushButton" name="schemeRemoveButton">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="toolTip">
        <string>Remove the selected scheme</string>
       </property>
       <property name="text">
        <string>Remove</string>
       </property>
       <property name="flat">
        <bool>false</bool>
       </property>
       <property name="icon">
        <iconset theme="delete">
         <normaloff/>
        </iconset>
       </property>
      </widget>
     </item>
     <item row="5" column="2">
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="5" column="3">
      <widget class="QPushButton" name="schemeImportButton">
       <property name="toolTip">
        <string>Import a color scheme from a file</string>
       </property>
       <property name="text">
        <string>&amp;Install from File...</string>
       </property>
       <property name="icon">
        <iconset theme="document-import">
         <normaloff/>
        </iconset>
       </property>
      </widget>
     </item>
     <item row="5" column="4">
      <widget class="QPushButton" name="schemeKnsButton">
       <property name="toolTip">
        <string>Get new color schemes from the Internet</string>
       </property>
       <property name="text">
        <string>Get &amp;New Color Schemes...</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0" colspan="6">
      <widget class="QListWidget" name="schemeList">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="iconSize">
        <size>
         <width>23</width>
         <height>16</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="1" column="0" colspan="6">
      <widget class="PreviewWidget" name="schemePreview" native="true">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
     <item row="6" column="0" colspan="3">
      <widget class="QCheckBox" name="applyToAlien">
       <property name="mouseTracking">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>Apply colors to &amp;non-Qt applications</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PreviewWidget</class>
   <extends>QWidget</extends>
   <header>previewwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>schemeList</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
